source/sync/configure/sync-settings.txt

```diff
@@ -74,9 +106,9 @@ For more information about modifying synced object schemas, see:
 :ref:`<synced-schema-overview>`.
 
 .. important:: Disable Development Mode for Production Apps
-   
-   Development mode is a development utility that is not suitable for 
-   production use. Make sure that you turn off Development Mode before you 
+
+   Development mode is a development utility that is not suitable for
```

Comment:
I know this already existed, but tiny edit for consistency. Thanks auto-formatting for making this catch my eye! 😆 
```suggestion
   Development Mode is a development utility that is not suitable for
```